RetailSuiteAccounting/UserInterface/01d_OpenItems.md

```diff
@@ -0,0 +1,226 @@
+
+  > [Info] The posting numbers must be sequential and are added by the system automatically.
+
+- *Date*  
+This column displays the posting date. The valid date format is DDMMYY.
+
+- *BP*  
+This column displays the posting period (month).
+
+- *Jnl*  
+This column displays the journal.
+
+- *Contra account*  
+This column displays the contra account number.
+
+- *Tax key*  
+This column displays the tax key linked to the contra account.
+
+- *Receipt1*  
+This column displays the receipt 1 value, usually the invoice number.
+
+  > [Info] The information provided in the *Receipt 1* field is essential, as the system identifies and groups together any subsequent postings taking this field as reference, for example for an invoice settlement or a credit note in case of refund.
+
+- *Receipt2*  
+This column displays the receipt 2 value, which can be any other reference number, such as the order number.
+
+- *Cst1*  
+This column displays the cost center number selected.
+
+  > [Info] This is only applicable if at least one cost center has been set up. For detailed information, see [COST ACCOUNTING](./02e_CostAccounting.md).
+
+- *Cst2*  
+This column displays the cost unit number selected.
+
+  > [Info] This is only applicable if at least one cost unit has been set up. For detailed information, see [COST ACCOUNTING](./02e_CostAccounting.md).
+
+- *Debit*  
+This column displays the debit amount of the selected account.
+
+- *Credit*  
+This column displays the credit amount of the selected account.
+
+- *(Unnamed column)*  
+This column displays a ![B](../../Assets/Icons/Beleg.png "[B]") icon if a receipt has been linked to the posting. When clicking the icon, the linked receipt is displayed in the *Attached document* window. For detailed information, see [Attached document](#attached-document).
+
+- *Text*  
+This column displays the posting text, if any has been entered.
+
+- *(Account number) (account name) [currency]*  
+This field introduces a list of all existing open items for this account.
+
+- *Balance (currency)*  
+This field displays the balance for each existing open item in the preceding account.  
+
+- *Amount (Amount 1 - Amount 2)*  
+The information provided in these fields refers to the postings list and is not related to the open items view. For detailed information, see [POSTINGS](./01a_Bookings.md).
+
+
+In the *Open items* tab, the input line serves the only purpose of selecting an account if no posting is selected in the *POSTINGS* tab. In this case, the input line fields are empty, except for the *Currency code* and the *Date* fields, which display the module basic currency and the first day of the current month respectively.
+
+> [Info] If a posting is selected in the *POSTINGS* tab, the posting details are displayed in the input line. The open items of the account selected in the *Account* field are displayed in the open items view, see [Open items view with account selection](#open-items-view-with-account-selection).
```

Comment:
wie in anderen Kapiteln sollte hier nur ein Verweis auf die Input line rein.
Die Info ist falsch - der Contra account wird angezeigt, wenn einer vorhanden ist, nicht der Account.